### DISCUSSION TITLE
Blocked response TTL - set to Zero seconds

### DISCUSSION THREAD

**Discussion by Dynamic5912** [AdguardTeam/AdGuardHome#7043] [General]
Are there any advantages or disadvantages to setting the Blocked response TTL to a number lower than 10 seconds, for example, 0 seconds?

**Comment by GrumpyGiuseppe** [comment]
I never changed it to anything else than 10.
But on our other discussion, I really have no clue what could cause that.
The best guess I can make is that your cache is stored on flash instead of ram, but that's only possible with a swap file.
![Screenshot 2024-05-31 192311](https://github.com/AdguardTeam/AdGuardHome/assets/65870540/99a037d3-7378-4f98-85cc-80d25e16eabb)
